### ISSUE TITLE
Allow using alt text to specifying whether highlights in embeds should be displayed/cleared

### ISSUE THREAD

**Issue by RyotaUshio** [RyotaUshio/obsidian-pdf-plus#11]
e.g. `![[file.pdf#page=1&selection=0,1,2,3|no highlight]]`

This can be problematic for accessibility reasons, but it should be fine as long as we use it for our private notes only.

**Comment by RyotaUshio** [comment]
Now that we can embed rectangular selections, this feature will no longer be important.
